plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/customization/CodeWhispererCustomizationDialog.kt

```diff
@@ -162,7 +161,12 @@ class CodeWhispererCustomizationDialog(
             lateinit var customizationComment: Row
             indent {
                 noCustomizationComment = row("") {
-                    rowComment(message("codewhisperer.custom.dialog.option.customization.description.no_customization", Q_CUSTOM_LEARN_MORE_URI))
+                    rowComment(
+                        message(
+                            "codewhisperer.custom.dialog.option.customization.description.no_customization",
+                            "https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/customizations.html"
```

Comment:
Should this be a const or do we expect this value to be updated?

Reply:
honestly it should probably just be in the localization bundle